### PR TITLE
adapt getParentTags to use conddblib instead of CondDBFW

### PR DESCRIPTION
**Pull request by mmusich**
Since https://github.com/cms-AlCaDB/AlCaTools/issues/11 is still open, I adapted the script to work without CondDFW using directly the conddblib commands.

### CODE REVIEW

**Comment by ghellwig** [comment]
@mmusich I see some similarities to the approach I have taken to circumvent this problem ;)

**Comment by mmusich** [comment]
@ghellwig, welcome back :)
If I recall correctly you are manager of this repo as well. If you are OK with changes then you can merge this. Anyway it's something I use from time to time to cross-check payloads. Maybe some more sophisticated version of this can become part of standard conddblib as distributed in `cmssw`.

File: scripts/getParentsTags.py [hunk 1]
```diff
@@ -60,41 +108,23 @@ def getCMSSWRelease( ):
...
-        'insertion', '',
-        )
+    #pp = pprint.PrettyPrinter(indent=4)
+    #pp.pprint(tags)
 
-    # Leave these alone for unquoted, tab-delimited record format.
-    head = '\t'.join(fldmap[0:len(fldmap):2]) + '\n'
+    #print tags
+    #for tag in tags:
+    #    print tag
```
Comment (ghellwig) [inline scripts/getParentsTags.py]: As this is not CMSSW, I don't mind to have these commented lines in the code, but in view of integrating it in CMSSW, we should get rid of it.

**Comment by ghellwig** [comment]
@mmusich done!

**Pull request merged**